hikari/impl/entity_factory.py

```diff
@@ -1739,21 +1739,25 @@ def deserialize_partial_message(  # noqa CFQ001 - Function too long
         if "stickers" in payload:
             stickers = [self._deserialize_sticker(sticker) for sticker in payload["stickers"]]
 
+        content = payload.get("content", undefined.UNDEFINED)
+        if content is not undefined.UNDEFINED:
+            content = content or None  # Default to None if content is an empty string
```

Comment:
Shouldn't an extra test be added to account for this case and lower the chance of any regressions in behaviour passing by unnoticed?

Reply:
Added :+1: